src/templates/dataGridTableRow.jsx

```diff
@@ -27,30 +40,42 @@ const propTypes = {
 };
 
 const defaultProps = {
-    handle: undefined,
+    classes: null,
+    handle: false,
     isClickable: undefined,
     rowProps: undefined,
 };
 
-class DataGridTableRow extends React.PureComponent {
+const styles = {
+    handleGrid: {
+        flexWrap: 'nowrap !important',
+    },
+};
+
+class DataGridTableRow extends React.Component {
     constructor() {
         super();
 
         this.onClick = this.onClick.bind(this);
     }
 
     onClick() {
-        const { isClickable, row, rowProps } = this.props;
-        const { onClick } = rowProps;
-        const isTextSelect = window.getSelection().toString();
+        const {
+            isClickable,
+            row,
+            rowProps: { onClick },
+        } = this.props;
 
-        if (isClickable && !isTextSelect && !_.isUndefined(onClick)) {
+        const isTextSelect = !isEmpty(window.getSelection().toString());
```

Comment:
Nitpack/NAB: Maybe `isTextSelected` instead of `isTextSelect`.

Reply:
Fixed.